src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -723,4 +723,9 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Property SourceProvider is null.</value>
     <comment>Don't localize 'SourceProvider'</comment>
   </data>
+  <data name="Error_AttemptingToPackSDKproject" xml:space="preserve">
+    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` or `msbuild -t:pack` to pack this project instead. You can override this behavior by setting the '{0}' environment variable to 'true'.</value>
+    <comment>0 - environment variable name. 
+Do not localize `SDK`, `dotnet pack`, msbuild -t:pack` and 'true'.</comment>
```

Comment:
```suggestion
    <value>The `pack` command for SDK-style projects is not supported, use `dotnet pack` or `msbuild -t:Pack` to pack this project instead. You can override this behavior by setting the '{0}' environment variable to 'true'.</value>
    <comment>0 - environment variable name. 
Do not localize `SDK`, `dotnet pack`, msbuild -t:Pack` and 'true'.</comment>
```

Even though MSBuild is case-insentive but the Target definition in NuGet Pack is defined as `Target Name="Pack"`. Hence, at least in product messages, we need to use the correct casing.